cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/EndpointLoader.kt

```diff
@@ -0,0 +1,51 @@
+package com.cognifide.cogboard.config
+
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import java.util.*
+
+class EndpointLoader(private val endpointsConfig: JsonArray, private val credentialsConfig: JsonArray, private val endpointId: String) {
+
+    fun asJson(withSensitiveData: Boolean): JsonObject {
```

Comment:
i prefer 2 methods here:
like: `asJson()` and `asSafeJson()`

```
fun asJson() = findById(endpointId, endpointsConfig)
fun asSafeJson() = asJson().attachUserPassword()
```

`attachUserPassword` as extension function

Reply:
shouldn't it be inversely?
`fun asSafeJson() = findById(endpointId, endpointsConfig)
fun asJson() = asJson().attachUserPassword()`
so `asSafeJson` return endpoint with `credentialsId`, but `asJson` returns endpoint with `user` and `password`

Reply:
Since those methods are located in `Endpoint**Loader**` class, I changed them into `load()` and `loadWithSensitiveData()` methods.